Dockerfile

```diff
@@ -171,13 +168,19 @@ RUN pip install --no-cache-dir "templateflow>=0.1.3,<0.2.0a0" && \
     find $TEMPLATEFLOW_HOME -type d -exec chmod go=u {} + && \
     find $TEMPLATEFLOW_HOME -type f -exec chmod go=u {} +
 
+# Installing bids-validator
+RUN npm install -g bids-validator@1.2.3
+
 # Installing FMRIPREP
 COPY . /src/fmriprep
 ARG VERSION
-# Force static versioning within container
-RUN echo "${VERSION}" > /src/fmriprep/fmriprep/VERSION && \
-    echo "include fmriprep/VERSION" >> /src/fmriprep/MANIFEST.in && \
-    pip install --no-cache-dir "/src/fmriprep[all]"
+# Simulate a git repository with one commit and a version
+RUN git config --global user.name null && \
+    git config --global user.email null && \
+    git -C /src/fmriprep init && \
+    git -C /src/fmriprep commit --allow-empty -m noop && \
+    git -C /src/fmriprep tag "${VERSION}"
```

Comment:
I'm honestly not sure why it doesn't mark these as dirty, but whatever. If it works, it works.